frontend/src/services/fetch/fetchers/applicationFetcher.ts

```diff
@@ -9,20 +9,30 @@ import { fetchApplicationWithMethod } from "./fetchers";
 
 export const handleStartApplication = async (
   competitionID: string,
+  token: string,
 ): Promise<ApplicationStartApiResponse> => {
+  const ssgToken = {
+    "X-SGG-Token": token,
+  };
   const response = await fetchApplicationWithMethod("POST")({
     subPath: `start`,
-    body: { competition_id: competitionID },
+    additionalHeaders: ssgToken,
+    body: { competition_id: competitionID, name: "test" },
```

Comment:
Competition now takes a "name" which we don't have a form for. This will be updated in this current epic.